src/site/content/en/lighthouse-seo/meta-description/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Document doesn't have a meta description
+title: Document doesn't have a `<meta>` description
```

Comment:
This should be left as-is. In this usage, _meta_ isn't a literal; it's an adjective for _description_. And the title changed in the lighthouse-seo cleanup anyway.